README.md

```diff
@@ -66,6 +66,7 @@ Clearance.configure do |config|
   config.sign_in_guards = []
   config.user_model = "User"
   config.parent_controller = "ApplicationController"
+  config.sign_in_on_password_reset = false
```

Comment:
I think this should be sufficient as the configuration option is quiet self-explanatory and there is additional information in the respective `lib/clerance/configuration.rb`.